crates/ibc/src/events.rs

```diff
@@ -153,13 +149,11 @@ pub enum IbcEventType {
     TimeoutOnClose,
     AppModule,
     Empty,
```

Comment:
I think we can also remove `IbcEventType::Empty` here.

Reply:
[2280d1f](https://github.com/cosmos/ibc-rs/pull/138/commits/2280d1f9844f1c3c53b21ae86fd2b6ba238e38c2)